pkg/collection/models/models.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+type CollectionResourceType string
+
+const (
+	ResourceCluster = CollectionResourceType("cluster")
```

Comment:
Is it possible to reuse common.ResourceCluster here?

Reply:
fixed